src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+  
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.big-search {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.big__img {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+  margin-top: 25px;
+}
+
+.small-search {
+  height: 42px;
+  width: 100%;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.small__img {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  margin-left: 13px;
+  margin-top: 15px;
+}
```

Comment:
Don't repeat yourself. Properties `position` and `width` are common for both bars, imgs. Move it to common class